resources/assets/less/bem/beatmapsets-search.less

```diff
@@ -60,22 +67,31 @@
     }
   }
 
+  &__filters {
+    display: flex;
+    justify-content: space-between;
+  }
+
   &__icon {
     position: absolute;
     right: 0;
     top: 0;
     height: 100%;
-    width: (@_icon-width + 20px);
```

Comment:
If you're wondering, this was originally intended so one day `math: strict` [option](http://lesscss.org/usage/#less-options-math) can be turned on.

(though apparently it's also needed for assigning variable which I haven't done)